File: CS/Reporting_how-to-create-a-label-report-at-runtime-t473792/LabelReportRuntime/CustomLabelReportModel.cs
namespace dxWinFormsSample {
    public class CustomLabelReportModel {
        public float LabelWidth { get; set; } 
        public float LabelHeight { get; set; } 
        public float VerticalPitch { get; set; }
        public float HorizontalPitch { get; set; }

        private System.Drawing.GraphicsUnit _Unit = System.Drawing.GraphicsUnit.Millimeter;
        public System.Drawing.GraphicsUnit MeasurementUnit { get { return _Unit; } set { _Unit = value; } }
        public float BottomMargin { get; set; }
        public float TopMargin { get; set; }
        public float LeftMargin { get; set; }
        public float RightMargin { get; set; }
        public int PaperKindID { get; set; }

    }
}

File: VB/Reporting_how-to-create-a-label-report-at-runtime-t473792/LabelReportRuntime/CustomLabelReportModel.vb
Namespace dxWinFormsSample
    Public Class CustomLabelReportModel
        Public Property LabelWidth() As Single
        Public Property LabelHeight() As Single
        Public Property VerticalPitch() As Single
        Public Property HorizontalPitch() As Single

        Private _Unit As System.Drawing.GraphicsUnit = System.Drawing.GraphicsUnit.Millimeter
        Public Property MeasurementUnit() As System.Drawing.GraphicsUnit
            Get
                Return _Unit
            End Get
            Set(ByVal value As System.Drawing.GraphicsUnit)
                _Unit = value
            End Set
        End Property
        Public Property BottomMargin() As Single
        Public Property TopMargin() As Single
        Public Property LeftMargin() As Single
        Public Property RightMargin() As Single
        Public Property PaperKindID() As Integer

    End Class
End Namespace
